src/services/identity/index.ts

```diff
@@ -3,12 +3,14 @@ import { PostgresIdentity } from './postgres.js'
 export { IIdentity } from './IIdentity.js'
 
 import * as dotenv from 'dotenv'
+import { Unauthorized } from './unauthorized.js'
 dotenv.config()
 
 export class Identity {
     private agent: LocalIdentity | PostgresIdentity
 
     static instance = new Identity().agent
+    static unauthorized = new Unauthorized()
 
     constructor() {
         if (process.env.ENABLE_EXTERNAL_DB === 'true') {
```

Comment:
@anikitinDSR 

Instead of this approach, can we add an optional argument to the constructor as `agentId?: string`

```
if (process.env.ENABLE_EXTERNAL_DB === 'true' ) {  
this.agent = agentId ? new PostgresIdentity(agentId) : Identity.unauthorized
 }
```
This will be useful once we integrate payments, as only authorized users can do payments. Hardcoding `Identity.unauthorized ` will be a problem 

Note we pass the agentId argument within PostgresIdentity